packages/gafl-webapp-service/src/pages/summary/contact-summary/route.js

```diff
@@ -143,7 +156,7 @@ const getContactDetails = permission => {
       getRow(
         'Licence details',
         getContactText(permission.licensee.preferredMethodOfReminder, permission.licensee, CONTACT_TEXT_NON_PHYSICAL),
-        CONTACT.uri,
+        addLanguageCodeToUri(request, CONTACT.uri),
```

Comment:
Need to test these Uris that the value being used is the ones returned from addLanguageCodeToUri. Also, I think this is the file Michael is refactoring - if not, we really need to refactor as it's insanely complex. I defy anyone to really know what's going on here!